client/client.go

```diff
@@ -162,6 +162,11 @@ type Client interface {
 	// this may differ from the initial Endpoints provided in the Config.
 	Endpoints() []string
 
+	// RestEndpoints resets the set of API endpoints used by Client to resolve
```

Comment:
Oh. OK. I feel Reset means set something again (not set it initially). I trust your english. So I will change it to set then.